proc_test.go

```diff
@@ -38,7 +73,7 @@ func newTestGProc(t *testing.T, w, h int, setup, draw func(p *Proc), fname strin
 func newTestProc(t *testing.T, w, h int, setup, draw func(p *Proc), fname string) *testProc {
 	t.Helper()
 
-	p := newProc(w, h)
+	p := newProc(w, h, testAppWindowFunc)
```

Comment:
```suggestion
	p := newProc(w, h)
	ctl := make(chan event.Event)
	p.newWindow = testAppWindow(ctl)
```

and a new field to `testProc`:
```go
type testProc struct {
    ...
    ctl chan event.Event
}
```
which will be initialized with that `ctl` variable above.